pospell.py

```diff
@@ -202,18 +209,18 @@ def quote_for_hunspell(text):
     against future changes in hunspell.
     """
     out = []
-    for line in text.split("\n"):
+    for line in text:
         out.append("^" + line if line else "")
     return "\n".join(out)
 
 
-def po_to_text(po_path, drop_capitalized=False):
+def po_to_text(drop_capitalized, po_path):
```

Comment:
Changing the order of the arguments of this function was not mandatory, you could have used a keyword argument to `partial`. But I think it's safe to bet that nobody used the function out of pospell, so I'm OK if you leave it as is too.

Reply:
I don´t use `partial` that often, so I didn't know/remember you could pass down keyword arguments down, sweet! I'll keep the old signature -- although this is a mid-size refactoring I really prefer it to be as less invasive as possible.